src/test/java/com/notably/logic/suggestion/SuggestionTestUtil.java

```diff
@@ -0,0 +1,108 @@
+import com.notably.testutil.TypicalBlockModel;
+
+/**
+ * Contains helper methods for testing suggestion feature.
+ */
+public class SuggestionTestUtil {
+    public static Model getModel() {
+        BlockModel blockModel = TypicalBlockModel.getTypicalBlockModel();
+        SuggestionModel suggestionModel = new SuggestionModelImpl();
+        ViewStateModel viewStateModel = new ViewStateModelImpl();
+        Model model = new ModelManager(blockModel, suggestionModel, viewStateModel);
+
+        return model;
+    }
+
+    public static AbsolutePath getToCs2103t() {
+        return TypicalBlockModel.PATH_TO_CS2103T;
+    }
+
+    public static List<SuggestionItem> getExpectedSugForCs2103tPathInput() {
+        SuggestionItem cs2103t = new SuggestionItemImpl(
+                TypicalBlockModel.PATH_TO_CS2103T.getStringRepresentation(), () -> {});
+        SuggestionItem cs2103tLect = new SuggestionItemImpl(
+                TypicalBlockModel.PATH_TO_CS2103T_LECTURES.getStringRepresentation(), () -> {});
+        SuggestionItem cs2103tTut = new SuggestionItemImpl(
+                TypicalBlockModel.PATH_TO_CS2103T_TUTORIALS.getStringRepresentation(), () -> {});
+        SuggestionItem cs2103tTut1 = new SuggestionItemImpl(
+                TypicalBlockModel.PATH_TO_CS2103T_TUTORIAL_1.getStringRepresentation(), () -> {});
+        SuggestionItem cs2103tTut2 = new SuggestionItemImpl(
+                TypicalBlockModel.PATH_TO_CS2103T_TUTORIAL_2.getStringRepresentation(), () -> {});
+
+        List<SuggestionItem> expectedSuggestions;
+        SuggestionItem[] items = new SuggestionItem[]{cs2103t, cs2103tLect, cs2103tTut, cs2103tTut1, cs2103tTut2};
+        expectedSuggestions = List.of(items);
+
+        return expectedSuggestions;
+    }
+
+    public static List<String> getExpectedInputsForCs2103tPathInput(String userInputWithoutPath) {
+        List<String> expectedInputs = new ArrayList<>();
+
+        expectedInputs.add(userInputWithoutPath + TypicalBlockModel.PATH_TO_CS2103T.getStringRepresentation());
+        expectedInputs.add(userInputWithoutPath + TypicalBlockModel.PATH_TO_CS2103T_LECTURES.getStringRepresentation());
+        expectedInputs.add(userInputWithoutPath
+                + TypicalBlockModel.PATH_TO_CS2103T_TUTORIALS.getStringRepresentation());
+        expectedInputs.add(userInputWithoutPath
+                + TypicalBlockModel.PATH_TO_CS2103T_TUTORIAL_1.getStringRepresentation());
+        expectedInputs.add(userInputWithoutPath
+                + TypicalBlockModel.PATH_TO_CS2103T_TUTORIAL_2.getStringRepresentation());
+
+        return expectedInputs;
+    }
+
+    /**
+     * Checks the correctness of the suggestions.
+     *
+     * @param expectedSuggestions The expected suggestions list.
+     * @param suggestions The actual list of suggestions.
+     */
+    public static void testSuggestions(List<SuggestionItem> expectedSuggestions, List<SuggestionItem> suggestions) {
```

Comment:
I guess naming this as `assertSuggestions` is more intuitive right? That way it resembles the name of `JUnit` methods!